benchmarks/cpp_micro_benchmarks.py

```diff
@@ -13,7 +13,7 @@
 
 OPTIONS = {
     "repetitions": {
-        "default": 6,
+        "default": 3,
```

Comment:
Had to cut this due to running more suites.

Reply:
Wouldn't it be better to keep the number of repetitions and reduce the repetition min time?

Reply:
Yes, it would. I'll try that experiment.

Reply:
I tried bumping this back up to 6 and the timing was fine. I'll merge that change in https://github.com/voltrondata-labs/benchmarks/pull/160.